subworkflows/nf-core/vcf_annotate_snpeff/main.nf

```diff
@@ -0,0 +1,28 @@
+//
+// Run SNPEFF to annotate VCF files
+//
+
+include { SNPEFF           } from '../../../modules/nf-core/snpeff/main.nf'
+include { TABIX_BGZIPTABIX } from '../../../modules/nf-core/tabix/bgziptabix/main.nf'
+
+workflow VCF_ANNOTATE_SNPEFF {
+    take:
+    vcf          // channel: [ val(meta), vcf ]
```

Comment:
Just checking in, did we decide for or against a `ch_` prefix for channels inputs?